openwisp_controller/config/templates/admin/device_group/change_form.html

```diff
@@ -12,7 +12,7 @@
         (function ($) {
             $(document).ready( function () {
                 window._relevantTemplateUrl = "{{ relevant_template_url | safe }}";
-                window._deviceGroup = true;
+                window._deviceGroupId = "{{ original.pk }}";
```

Comment:
We were already setting the `_devieGroup` property here to determine whether the relevant_template,js script was loading from device page or device group page. I have updated the logic to pass the device group ID to the JS. 

@nemesifier if you prefer an HTML element, I would update the code accordingly.